arbiter-core/src/manager.rs

```diff
@@ -1,3 +1,9 @@
+//! The `manager` module provides structures and functionalities for managing
+//! environments that house simulations.
+//!
+//! It centralizes operations such as creating/adding environments as well as
+//! starting, pausing, and stopping a chosen environment.
+
 #![warn(missing_docs, unsafe_code)]
 
 // TODO: Check the publicness of all structs and functions.
```

Comment:
Do we want to do this in this PR or later?

Reply:
I'm confused, we certainly want the manager to manage environments. That's the only public API after all.

Reply:
I think we should do this in the `feat/middleware` PR yes.

Some of it was done in this PR too.